implement next/prev buttons

saadamir3180 opened the pull request:
implement Next/PrevItem in Blog & Portfolio, added disabled class in template/sections/_portfolio.scss to disable next/Prev for no projects/blogs

njmulsqb commented:
When I hit "Next Post" and there's no next post it opens the same post again, this button should be disabled if there's no next post 
<img width="381" alt="image" src="https://github.com/user-attachments/assets/02b3c84b-43c4-4918-b9eb-404747a6951d">
These next and previous buttons are not working
<img width="719" alt="image" src="https://github.com/user-attachments/assets/f0fddfc7-2fd8-4f40-870b-5032f6ef05db">

Please do basic testing yourself before asking for review.

saadamir3180 commented:
> When I hit "Next Post" and there's no next post it opens the same post again, this button should be disabled if there's no next post <img alt="image" width="381" src="https://private-user-images.githubusercontent.com/47940898/376451657-02b3c84b-43c4-4918-b9eb-404747a6951d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5NjgwNDUsIm5iZiI6MTcyODk2Nzc0NSwicGF0aCI6Ii80Nzk0MDg5OC8zNzY0NTE2NTctMDJiM2M4NGItNDNjNC00OTE4LWI5ZWItNDA0NzQ3YTY5NTFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE1VDA0NDkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVhZDEzYjg0YjZhNTRkZmQ3ODFkODViMjk2OTgzYWZjZmY0MDg0MzA4OWU2YTIwMGZmYmE5YzcwOGY0ZTM1YjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.P8YjEwuXU5fwxNuLNCs3XdfhqWnFHOgZIXROmTMrJlw"> These next and previous buttons are not working <img alt="image" width="719" src="https://private-user-images.githubusercontent.com/47940898/376451736-f0fddfc7-2fd8-4f40-870b-5032f6ef05db.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5NjgwNDUsIm5iZiI6MTcyODk2Nzc0NSwicGF0aCI6Ii80Nzk0MDg5OC8zNzY0NTE3MzYtZjBmZGRmYzctMmZkOC00ZjQwLTg3MGItNTAzMmY2ZWYwNWRiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE1VDA0NDkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZmNThkMTMwZjI0MDI3MGE4MGJjODUwOGQyOWFjNDJkZjQ2OGZkNWY5YjU1NzRiYjNhMWJjYjNmNTUyNDlhYWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Cwh_wu7GdUS0xK7ULYMsug7F_3omy6qhC5FqGA5PgA0">
> 
> Please do basic testing yourself before asking for review.

The `Next Post` button is working correctly, why you see the repeated blog was due to the redundant data in the blog file.
I also added relevant data for blog, to avoid confusion.

 The `Next/Previous` Buttons are now implemented.

njmulsqb commented:
Build is failing: https://pastebin.com/NG6uaWbf
